### ISSUE TITLE
看到你的作品后对你的病情高度重视

### ISSUE THREAD

**Issue by maimos** [kelseyhightower/nocode#4844]
我是北京大学第六医院副主任医师，我院在看到你的作品后，对你的病情高度重视，立刻安排我与你取得联系！ 精神病是一种可治可防的病，只要你积极配合治疗，一定可以完全治愈，最后希望你树立正确的品行，建立信心，战胜病魔！ 祝 你早日康复。

**Comment by johannesrld** [comment]
yeah I lost my left testicle after an unfortunate smelting accident back in 1973, can you help with that doctor?

**Comment by MaybeQHL** [comment]
> yeah I lost my left testicle after an unfortunate smelting accident back in 1973, can you help with that doctor?

wow, can you show some images?
